ci-operator/step-registry/openshift-qe/orion/openshift-qe-orion-commands.sh

```
@@ -55,6 +55,14 @@ case "$ES_TYPE" in
     ;;
 esac
 
+if [[ -f "${CLUSTER_PROFILE_DIR}/jira-api-key" ]]; then
+    export JIRA_TOKEN=$(<"${CLUSTER_PROFILE_DIR}/jira-api-key")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)